msgpack_numpy.py

```diff
@@ -154,37 +159,33 @@ def __init__(self, file_like=None, read_size=0, use_list=None,
                                            max_buffer_size=max_buffer_size,
                                            ext_hook=ext_hook)
 
-def pack(o, stream, default=encode, **kwargs):
+def pack(o, stream, **kwargs):
     """
     Pack an object and write it to a stream.
     """
 
```

Comment:
actually, I think msgpack_numpy would not need to patch `pack()`, `packb()`, `unpack()` and `unpackb()` anymore.

Reply:
Well, I just had a quick look at the msgpack code so... maybe not :-(